chat/chat.js

```diff
@@ -0,0 +1,32 @@
+const messageContainer = document.querySelector("#message-container")
+
+function addFirstUserMessage() {
+    const messageElement = createMessageElement("first-input");
+    messageElement.className += " main-user";
+    displayMessageInContainer(messageElement)
+    document.getElementById("first-input").value = "";
+}
+
+function addSecondUserMessage() {
```

Comment:
addFirstUserMessage and addSecondUserMessage break the DRY rule, try to unify this functions (maybe use some specific params for function)

For example smth like: addUserMessage(messageElementSelector, isMainUser)